clusterloader2/testing/density/config.yaml

```diff
@@ -1,250 +1,2 @@
-# ASSUMPTIONS:
-# - Underlying cluster should have 100+ nodes.
-# - Number of nodes should be divisible by NODES_PER_NAMESPACE (default 100).
-
-#Constants
-{{$DENSITY_RESOURCE_CONSTRAINTS_FILE := DefaultParam .DENSITY_RESOURCE_CONSTRAINTS_FILE ""}}
-{{$NODE_MODE := DefaultParam .NODE_MODE "allnodes"}}
-{{$NODES_PER_NAMESPACE := DefaultParam .NODES_PER_NAMESPACE 100}}
-{{$PODS_PER_NODE := DefaultParam .PODS_PER_NODE 30}}
-{{$DENSITY_TEST_THROUGHPUT := DefaultParam .DENSITY_TEST_THROUGHPUT 20}}
-{{$SCHEDULER_THROUGHPUT_THRESHOLD := DefaultParam .CL2_SCHEDULER_THROUGHPUT_THRESHOLD 0}}
-# LATENCY_POD_MEMORY and LATENCY_POD_CPU are calculated for 1-core 4GB node.
-# Increasing allocation of both memory and cpu by 10%
-# decreases the value of priority function in scheduler by one point.
-# This results in decreased probability of choosing the same node again.
-{{$LATENCY_POD_CPU := DefaultParam .LATENCY_POD_CPU 100}}
-{{$LATENCY_POD_MEMORY := DefaultParam .LATENCY_POD_MEMORY 350}}
-{{$MIN_LATENCY_PODS := 500}}
-{{$MIN_SATURATION_PODS_TIMEOUT := 180}}
-{{$ENABLE_CHAOSMONKEY := DefaultParam .ENABLE_CHAOSMONKEY false}}
-{{$ENABLE_PROMETHEUS_API_RESPONSIVENESS := DefaultParam .ENABLE_PROMETHEUS_API_RESPONSIVENESS false}}
-{{$ENABLE_SYSTEM_POD_METRICS:= DefaultParam .ENABLE_SYSTEM_POD_METRICS true}}
-{{$USE_SIMPLE_LATENCY_QUERY := DefaultParam .USE_SIMPLE_LATENCY_QUERY false}}
-{{$ENABLE_RESTART_COUNT_CHECK := DefaultParam .ENABLE_RESTART_COUNT_CHECK false}}
-{{$RESTART_COUNT_THRESHOLD_OVERRIDES:= DefaultParam .RESTART_COUNT_THRESHOLD_OVERRIDES ""}}
-#Variables
-{{$namespaces := DivideInt .Nodes $NODES_PER_NAMESPACE}}
-{{$podsPerNamespace := MultiplyInt $PODS_PER_NODE $NODES_PER_NAMESPACE}}
-{{$totalPods := MultiplyInt $podsPerNamespace $namespaces}}
-{{$latencyReplicas := DivideInt (MaxInt $MIN_LATENCY_PODS .Nodes) $namespaces}}
-{{$totalLatencyPods := MultiplyInt $namespaces $latencyReplicas}}
-{{$saturationDeploymentTimeout := DivideFloat $totalPods $DENSITY_TEST_THROUGHPUT | AddInt $MIN_SATURATION_PODS_TIMEOUT}}
-# saturationDeploymentHardTimeout must be at least 20m to make sure that ~10m node
-# failure won't fail the test. See https://github.com/kubernetes/kubernetes/issues/73461#issuecomment-467338711
-{{$saturationDeploymentHardTimeout := MaxInt $saturationDeploymentTimeout 1200}}
-
+# TODO(https://github.com/kubernetes/perf-tests/issues/1007): Get rid of this file
```

Comment:
There is also high-density test, which relies on it.
Medium-term, we should modify the new load to support it (e.g. instead of creating 2 deployments, create N), but short-term, maybe let's leave this test.

Reply:
Sent you PRs to address it